src/Microsoft.Graph/Generated/model/AuthenticationMethodResetPasswordRequestBody.cs

```diff
@@ -26,11 +26,5 @@ public partial class AuthenticationMethodResetPasswordRequestBody
         [JsonPropertyName("newPassword")]
         public string NewPassword { get; set; }
     
-        /// <summary>
```

Comment:
Need to validate the change before merging as it is breaking.

Reply:
According to 
- https://github.com/microsoftgraph/microsoft-graph-docs/pull/18789
- https://dev.azure.com/msazure/One/_git/AD-AggregatorService-Workloads/pullrequest/6941748

This removal is a fix as a this parameter was accidentally pushed to v1.0 production and only works for a handful of first party apps.

From this it looks that this is not a breaking change.

cc @baywet, @ramsessanchez , @Ndiritu , @SilasKenneth ,

Reply:
thanks for sharing the information here!